RELEASE.md

```diff
@@ -0,0 +1,5 @@
+release type: patch
+
+Integrate the `strawberry-graphql-django` package into Strawberry core so that it's
+possible to use the Django extension package directly through the `strawberry.django`
+namespace.
```

Comment:
```suggestion
This release tntegrates the `strawberry-graphql-django` package into Strawberry core so that it's possible to use the Django extension package directly via the `strawberry.django`
namespace.

You still need to install `strawberry-graphql-django` if you want to use the extended django support.

See: https://github.com/strawberry-graphql/strawberry-graphql-django
```

Reply:
RELEASE.md updated